src/Main.elm

```diff
@@ -189,14 +196,9 @@ update msg model =
 
                 cellsToArm : List Cell
                 cellsToArm =
-                    List.map
+                    List.filterMap
                         (\index ->
-                            case Array.get index available of
-                                Just cell ->
-                                    cell
-
-                                Nothing ->
-                                    Debug.crash "nah"
+                            Array.get index available                            
```

Comment:
nice!

Reply:
I had to rename the temp variables because the compiler no longer allows it.